astropy/modeling/functional_models.py

```diff
@@ -91,6 +91,32 @@ class Gaussian1D(Fittable1DModel):
     amplitude = Parameter(default=1)
     mean = Parameter(default=0)
     stddev = Parameter(default=1)
+    _bbox = 'auto'
+
+    @property
+    def bbox(self):
+        '''
+        Returns the bounding box limits (x_low, x_high)
+        Default is (-5 * x_stddev, 5 * x_stddev)
+
+        model.bbox = 'auto' will reset the default.
+        '''
+        if self._bbox == 'auto':
```

Comment:
Can a property have an `default` attribute like parameters do?

You can get the default value of x_0 with model.x_0.default, so I think there ought to be an equivalent for this property.